mcp_server.py

```
@@ -227,6 +227,83 @@ async def list_devices() -> Dict[str, Any]:
+        
+    except httpx.HTTPError as e:
+        logger.error(f"HTTP error adding device {hostname}: {e}")
+        if e.response.status_code == 409:
+            raise Exception(f"Device with hostname '{hostname}' already exists")
+        raise Exception(f"Failed to add device: {str(e)}")
+    except Exception as e:
+        logger.error(f"Error adding device {hostname}: {e}")
+        raise Exception(f"Failed to add device: {str(e)}")
+
+
+async def remove_device(hostname: str) -> Dict[str, Any]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)